reference/src/packages.md

```diff
@@ -157,65 +152,60 @@ MoveStdlib = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/s
 address_to_be_filled_in = "0x101010101"
 ```
 
-Most of the sections in the package manifest are self explanatory, but named
-addresses can be a bit difficult to understand so we examine them in more
-detail in [Named Addresses During
-Compilation](#named-addresses-during-compilation), but before that we'll first
-take a look at the `Move.lock` file and what it contains.
+Most of the sections in the package manifest are self explanatory, but named addresses can be a bit
+difficult to understand so we examine them in more detail in
+[Named Addresses During Compilation](#named-addresses-during-compilation), but before that we'll
+first take a look at the `Move.lock` file and what it contains.
```

Comment:
IMO this is a confusion transition. I would consider swapping the two sections (since the lockfile doesn't appear to be mentioned in the address discussion) and rewrite this as:

> Most of the sections in the package manifest should be self explanatory, with the exception of named addresses which are explained in the 
[next section](#named-addresses-during-compilation). After being built, the `Move.toml` and configuration settings are stored in the [`Move.lock` file](#move.lock).

Reply:
I'll give it a shot!

Reply:
I like this a lot more than the current layout, so 👍🏻 from me on moving this section around.